backend/src/test/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceAsyncTest.java

```diff
@@ -122,6 +121,90 @@ void validPerson_savedToDatabase() throws IOException, ExecutionException, Inter
             Map.of("patients_url", "https://simplereport.gov/patients?facility=null"));
   }
 
+  @Test
+  void validPersonWithRequiredOnlyFields_savedToDatabase()
```

Comment:
thanks for these tests! do you think we could compare the # of columns to the size of [requiredFields](https://github.com/CDCgov/prime-simplereport/blob/25c49d09e87336d671ab8c9f779767676284d5b9/backend/src/main/java/gov/cdc/usds/simplereport/api/model/filerow/PatientUploadRow.java#L62-L76)? or add a comment above requiredFields so that whenever someone changes that they should also update this test?

or maybe it would be better to mock [`isRequired`](https://github.com/CDCgov/prime-simplereport/blob/25c49d09e87336d671ab8c9f779767676284d5b9/backend/src/main/java/gov/cdc/usds/simplereport/api/model/filerow/PatientUploadRow.java#L115)

Reply:
I'll add a change to check that the CSV has the same required fields!